lcoe_calculator/extractor.py

```diff
@@ -30,14 +30,16 @@ class Extractor(AbstractExtractor):
     tax_credits_sheet = 'Tax Credits'
 
     def __init__(self, data_workbook_fname: str, sheet_name: str, case: str, crp: CrpChoiceType,
-                 scenarios: List[str], base_year: int):
+                 scenarios: List[str], base_year: int, tax_credit_case : str = None):
```

Comment:
This is starting to stretch my OOP knowledge, but if tax_credit_case is only used in the child class, I think it can (and should) be left out of the parent constructor. Does that sound right? Also use Optional[str] for the tcc type.